### ISSUE TITLE
2023_jul_epic #story_h: Posting IIR Data Scientist GS-13 DEU hiring

### ISSUE THREAD

**Issue by emilymetcalf114** [lzim/teampsd#2857]
@emilymetcalf114 @epearman @lzim @hthrmarsh 
refer to #crossref #2871 for core competencies 

- [x] Letter – Eligible for TW. Just need Lindsey to ONLY e-sign below her name an return.
- [x] New Staff Checklist – complete and return
- [x] VA0938 JA and Category Rating- meeting with Randy to discuss next steps **wk5 jan epic hiring_workflow**

**Comment by hthrmarsh** [comment]
Lindsey
From @lzim 
- [ ] My understanding of the next steps is to get that position into great alignment with our needs for the role than Pearman, Emily K. was able to provide with her first step.
- [ ] Use the wk2_2023_mar_epic hiring_workflow to review competencies

**Comment by emilymetcalf114** [comment]
#cross-ref #2871

**Comment by lzim** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-PSD-Workspace-5e4ac9bd8d388dfba6d9aef5/issues/lzim/teampsd/2857?planning-poker) with Zenhub @emilymetcalf114 @epearman

**Comment by lzim** [comment]
- [ ] @hthrmarsh review the Job Analysis Competency Ratings
- [ ] @hthrmarsh review DataOps Manager P.D. in **training_workflow**
- [ ] @hthrmarsh review the [SAFe 6.0 ](https://scaledagileframework.com/#)
- [ ] @lzim Data Scientist/Statistician P.D.

_User Pain Point_: 
DataOps Manager needs 
C++ (computer science language) 
API
Government Cloud Computing support to automatic Data UI Updates

![image.png](https://images.zenhubusercontent.com/5e4ac859b8781e56a5b7d6a1/d7756597-cbb3-423a-94ba-f99f6d0a48a9)

**Comment by hthrmarsh** [comment]
@lzim 

Hi Lindsey - getting clarity on the "ask" still. 

1) Is this a **DEU recruitment** for this and Randy is to help with the DEU recruitment **using the newly classified GS-13 Data Scientist PD**? In this case we'd want to pay attention to the Job Analysis competencies since if a DEU recruitment (so not a direct hire) then the Job Analysis competencies need to come from the established/approved PD. 
2) Or are you wanting to put together a **new Position Description at the VA?** In which case we might bring in SAFe if working on a new PD. 
3) Or is this a new direct hire and we're just working on a job ad?

**Comment by emilymetcalf114** [comment]
@hthrmarsh 
This is a DEU recruitment using the newly classified GS-13 Data Scientist PD. Emily P reviewed the list of competencies that Randy provided via email and documented on #cross-ref #2871

**Comment by hthrmarsh** [comment]
Submitted draft copies of requested paperwork to Randy Gieben 3/21/23.

**Comment by hthrmarsh** [comment]
Will make a new card for the MSA ad/hire process. 
- [ ] @hthrmarsh to work on draft ad pursuing N/C hire as well. 
- [ ] Lindsey to review job ad draft
- [ ] Heather to post ad.

**Comment by hthrmarsh** [comment]
@lzim 

This form is for a **consultation only** with the new HRMACS group. I recommend you s**ign this form so they can get the consult mtg** (which is at least a couple weeks out). **In the meantime, you can review the competencies. See #2871** 

**File location here (just need to sign it):**

![image](https://user-images.githubusercontent.com/126210778/232328205-7ad0a1fe-fc1b-48f8-a835-a9ddb3571a72.png)

**Comment by hthrmarsh** [comment]
HRMACS Competitive Recruitment Request Form sent to Randy 4/19/23

**Comment by lzim** [comment]
@hthrmarsh 

Thanks for all your help with this!
I sent the Job Analysis form to the DEU folks.

_I think we're waiting on them now, right?
Should we just pull you off this one too?_

Thanks again!

**Comment by hthrmarsh** [comment]
@lzim yes and confirm the specialized experience statement. Then it's with them to send you (and Randy) the documents back.

**Comment by hthrmarsh** [comment]
- [ ] Randy Gieben to submit to MSS 
![image.png](https://images.zenhubusercontent.com/6408f6bee4d68058a5b2cf6b/eb67b728-6595-46f1-86bb-30917809d72b)
